server/db/user-migration.sql

```diff
@@ -0,0 +1,135 @@
+DROP TABLE IF EXISTS users;
+-- Create Users Table
+CREATE TABLE users (
+   id serial PRIMARY KEY,
+   userName VARCHAR(50) NOT NULL,
+   email VARCHAR(50) UNIQUE NOT NULL,
+   role VARCHAR(50) NOT NULL,
+   createdById INT NOT NULL,
+   createdAt TIMESTAMP NOT NULL DEFAULT NOW(),
+   updatedById INT NOT NULL,
+   updatedAt TIMESTAMP NOT NULL DEFAULT NOW()
```

Comment:
I don't think we need this information on user
instead we can add the country of the user
```suggestion
 country VARCHAR(50) NOT NULL
```